subscription.go

```diff
@@ -0,0 +1,219 @@
+package opcua
+
+import (
+	"context"
+	"math"
+	"time"
+
+	"github.com/gopcua/opcua/ua"
+)
+
+const (
+	DefaultSubscriptionMaxNotificationsPerPublish = 10000
+	DefaultSubscriptionLifetimeCount              = 10000
+	DefaultSubscriptionMaxKeepAliveCount          = 3000
+	DefaultSubscriptionInterval                   = 100 * time.Millisecond
+	DefaultSubscriptionPriority                   = 0
+)
+
+type Subscription struct {
+	SubscriptionID            uint32
+	RevisedPublishingInterval float64
```

Comment:
Can we make this a `time.Duration` just like `Interval` below?

Reply:
Yes, we can. `float64` is the type defined for response by the spec. But I see no problem converting it  for ease of use